.drone.star

```diff
@@ -521,110 +521,48 @@ def accountsUITests(ctx, phoenixBranch, phoenixCommitId, storage):
       'os': 'linux',
       'arch': 'amd64',
     },
-    'steps': [
-      {
-        'name': 'build-ocis',
-        'image': 'webhippie/golang:1.13',
-        'pull': 'always',
-        'commands': [
-          'cd ocis',
-          'make build',
-          'mkdir -p /srv/app/ocis/bin',
-          'cp bin/ocis /srv/app/ocis/bin',
-        ],
-        'volumes': [
-          {
-            'name': 'gopath',
-            'path': '/srv/app'
-          },
-        ]
-      },
-      {
-        'name': 'ocis-server',
-        'image': 'webhippie/golang:1.13',
-        'pull': 'always',
-        'detach': True,
-        'environment' : {
-          #'OCIS_LOG_LEVEL': 'debug',
-          'STORAGE_HOME_DRIVER': '%s' % (storage),
-          'STORAGE_STORAGE_OCIS_ROOT': '/srv/app/tmp/ocis/storage/users',
-          'STORAGE_STORAGE_LOCAL_ROOT': '/srv/app/tmp/ocis/reva/root',
-          'STORAGE_STORAGE_OWNCLOUD_DATADIR': '/srv/app/tmp/ocis/owncloud/data',
-          'STORAGE_STORAGE_OWNCLOUD_REDIS_ADDR': 'redis:6379',
-          'STORAGE_OIDC_ISSUER': 'https://ocis-server:9200',
-          'STORAGE_LDAP_IDP': 'https://ocis-server:9200',
-          'PROXY_OIDC_ISSUER': 'https://ocis-server:9200',
-          'STORAGE_HOME_EXPOSE_DATA_SERVER': 'true',
-          'STORAGE_HOME_DATA_SERVER_URL': 'http://ocis-server:9155/data',
-          'STORAGE_DATAGATEWAY_PUBLIC_URL': 'https://ocis-server:9200/data',
-          'STORAGE_FRONTEND_PUBLIC_URL': 'https://ocis-server:9200',
-          'PHOENIX_WEB_CONFIG': '/drone/src/accounts/ui/tests/config/drone/ocis-config.json',
-          'KONNECTD_IDENTIFIER_REGISTRATION_CONF': '/drone/src/accounts/ui/tests/config/drone/identifier-registration.yml',
-          'KONNECTD_ISS': 'https://ocis-server:9200',
-          'ACCOUNTS_STORAGE_DISK_PATH': '/srv/app/tmp/ocis-accounts', # Temporary workaround, don't use metadata storage
-        },
-        'commands': [
-          'mkdir -p /srv/app/tmp/reva',
-          # First run settings service because accounts need it to register the settings bundles
-          '/srv/app/ocis/bin/ocis settings &',
-
-          # Wait for the settings service to start
-          "while [[ \"$(curl -s -o /dev/null -w ''%{http_code}'' localhost:9190)\" != \"404\" ]]; do sleep 2; done",
-
-          # Now start the accounts service
-          '/srv/app/ocis/bin/ocis accounts &',
-
-          # Wait for the accounts service to start
-          "while [[ \"$(curl -s -o /dev/null -w ''%{http_code}'' localhost:9181)\" != \"404\" ]]; do sleep 2; done",
-
-          # Now run all the ocis services except the accounts and settings because they are already running
-          '/srv/app/ocis/bin/ocis server',
-        ],
-        'volumes': [
-          {
-            'name': 'gopath',
-            'path': '/srv/app'
-          },
-        ]
-      },
+    'steps':
+      generate('ocis') +
+      build() +
+      ocisServer(storage) + [
```

Comment:
Can we really start the full server stack?

Reply:
why not? I added the accounts UI to the web config that is being used for the ocisServer. With that it should be fit for use.

Reply:
streamlines the setup, fine